controller/src/main/resources/json/stock/patch-collection-shoe-with-no-valid-json-payload-capacity-exception.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Being used by tests, they should be in src/test/resources and not main.

Reply:
fix and is not push yet.